vertica.py

```diff
@@ -0,0 +1,28 @@
+# pip install vertica-python
+
+from vertica_python import connect
+import json
+connection = connect({
+    'host': '.mozilla.com',
+    'port': 5433,
+    'user': '',
+    'password': '',
+    'database': ''
```

Comment:
You'll need to read this information from a file, not embed it into the script. Otherwise we can't deploy the script without local modifications.